packages/components/src/components/post-header/post-header.tsx

```diff
@@ -57,6 +86,47 @@ export class PostHeader {
     if (!this.mobileMenuExtended) await this.mobileMenuAnimation.finished;
   }
 
+  // Get all the focusable elements in the post-header mobile menu
+  private getFocusableElements() {
+    // Get elements in the correct order (different as the DOM order)
+    const focusableEls = [
+      ...Array.from(this.host.querySelectorAll('.list-inline:not([slot="meta-navigation"]) > li')),
+      ...Array.from(
+        this.host.querySelectorAll(
+          'nav > post-list > div > post-list-item, post-mainnavigation > .back-button, post-megadropdown-trigger',
+        ),
+      ),
+      ...Array.from(
+        this.host.querySelectorAll(
+          '.list-inline[slot="meta-navigation"] > li, post-language-option',
+        ),
+      ),
+    ];
+
+    // Add the main toggle menu button to the list of focusable children
+    const focusableChildren = [
+      this.host.querySelector('post-togglebutton'),
+      ...focusableEls.flatMap(el => Array.from(getFocusableChildren(el))),
+    ];
+
+    this.firstFocusableEl = focusableChildren[0];
+    this.lastFocusableEl = focusableChildren[focusableChildren.length - 1];
+  }
+
+  private keyboardHandler(e: KeyboardEvent) {
+    if (e.key === 'Tab') {
+      if (e.shiftKey && document.activeElement === this.firstFocusableEl) {
+        // If back tab (Tab + Shift) and first element is focused, focus goes to the last element of the megadropdown
+        e.preventDefault();
```

Comment:
This should only be done if the menu is open, otherwise it's impossible to go back to the logo and skiplinks:

![image](https://github.com/user-attachments/assets/3927c5a9-d5ed-4e41-9626-72ba7fd69a3a)

Pressing [shift] + [tab] in this situation locks focus to the menu button.